documentation/faq/collections.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Preparation for Converting Your Role to the Collections Format
+---
+
+If you are developing a role in linux-system-roles, you may find your question already answered here.
+
+## Module and Module_util name
```

Comment:
module_utils

Reply:
Thanks, @richm. Yeah, I was wondering whether they should be singular or plural... How about Module?

Reply:
I think module is ok